From f4ec87cb2b84e433559caf117ef4af2869d4198e Mon Sep 17 00:00:00 2001
From: Torge Matthies <openglfreak@googlemail.com>
Date: Mon, 25 Apr 2022 05:24:26 +0200
Subject: [PATCH 08/16] Revert "server: Clear the
 MOUSEEVENTF_(ABSOLUTE|VIRTUALDESK) flags."

This reverts staging patch patches/user32-rawinput-mouse-experimental/0001-server-Clear-the-MOUSEEVENTF_-ABSOLUTE-VIRTUALDESK-f.patch
---
 server/queue.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/queue.c b/server/queue.c
index 11111111111..11111111111 100644
--- a/server/queue.c
+++ b/server/queue.c
@@ -2167,7 +2167,7 @@ static int queue_mouse_message( struct desktop *desktop, user_handle_t win, cons
         msg_data = &raw_msg.data;
         msg_data->info                = input->mouse.info;
         msg_data->size                = sizeof(*msg_data);
-        msg_data->flags               = flags & ~(MOUSEEVENTF_ABSOLUTE | MOUSEEVENTF_VIRTUALDESK);
+        msg_data->flags               = flags;
         msg_data->rawinput.type       = RIM_TYPEMOUSE;
         msg_data->rawinput.mouse.x    = x - desktop->shared->cursor.x;
         msg_data->rawinput.mouse.y    = y - desktop->shared->cursor.y;
-- 
0.0.0

